core/src/main/scala/io/finch/EndpointResult.scala

```diff
@@ -21,6 +23,8 @@ import scala.concurrent.duration.{Duration => ScalaDuration}
  */
 sealed abstract class EndpointResult[F[_], +A] {
 
+  val Top = ScalaDuration(Long.MaxValue, TimeUnit.NANOSECONDS)
```

Comment:
Can we use `Duration.Inf` instead?

Reply:
I used finite duration as a easy way to translate to `twitter.Duration`. As infinite duration would not have value/length. I fixed this too.